src/processing/resources/index.js

```diff
@@ -37,8 +38,8 @@ export async function process (ctx, customCharset) {
             var urlReplacer = getResourceUrlReplacer(ctx);
             var processed   = processors[i].processResource(decoded, ctx, charset, urlReplacer);
 
-            if (processed === null)
-                return null;
+            if (pageProcessor.isRestartProcessingSymbol(processed))
```

Comment:
Why not just make `RESTART_PROCESSING` field of the `pageProcessor` and just compare it  directly here, IMHO it will be much clear